docs/examples/use_cases/tensorflow/efficientdet/docker/docker_build.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--tag", default="dali-effdet", help="Tag for ")
+args = parser.parse_args()
+
+# We copy tfrecord2idx to
+os.system(" cp ../../../../../../tools/tfrecord2idx ../__docker_tfrecord2idx")
```

Comment:
Hmm.... This would fail miserably when launched from a different path:
```python  docs/examples/use_cases/tensorflow/efficientdet/docker/docker_build.py```
Can we use a path relative to the script source?

Reply:
I think we can remove this whole file. It's not used by the example or the QA test.

Reply:
I removed that file.